packages/fuels-signers/src/wallet.rs

```diff
@@ -467,13 +510,26 @@ impl WalletUnlocked {
             .inputs()
             .iter()
             .any(|input| matches!(input, Input::CoinSigned { .. }));
+
         if !is_using_coins && new_base_amount == 0 {
             new_base_amount = MIN_AMOUNT;
         }
 
+        // This is a temporary solution till we get update on coins_to_spend function
```

Comment:
Can we put a note in the `add_fee_coins` docstring about using `InputMessages` to cover fees in case there aren't enough regular assets?